src/assets/styles/app.css

```diff
@@ -176,9 +176,12 @@ main {
 main header h1 {
     font-size: 3rem;
     font-weight: 800;
+    -webkit-hyphens: auto;
```

Comment:
I can't remember now, are we still using postCSS? If so, you may not need the vendor prefixed versions.